Makefile

```diff
@@ -11,51 +15,46 @@ ifndef GOARCH
 GOARCH := $(shell go env GOARCH)
 endif
 
-default: test
+MODULE_ROOT := github.com/temporalio/temporal
+BUILD := ./build
+COLOR := "\e[1;36m%s\e[0m\n"
```

Comment:
very fancy, but not too fancy! ; )

Reply:
This is my wife's favourite color.

Reply:
"\e[1;36m%s\e[0m\n" Is a great color! ;)